lib/pbench/cli/agent/commands/timestamp.py

```diff
@@ -0,0 +1,13 @@
+"""pbench-log-timestamp"""
+
+import time
+
+import click
+
+
+@click.command()
+def main():
+    """ Add timestamp to user input"""
+    stdin = click.get_text_stream("stdin").read()
```

Comment:
This needs to be a generator so that `stdin` can be a pipe and each line read from the pipe is timestamped and written out.

Further, I think we'll want to make sure it is not using buffered I/O so that folks can inspect the output easily as it is generated.